pkg/odo/cli/create/binding.go

```diff
@@ -0,0 +1,119 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	ktemplates "k8s.io/kubectl/pkg/util/templates"
+
+	"github.com/redhat-developer/odo/pkg/odo/cmdline"
+	"github.com/redhat-developer/odo/pkg/odo/genericclioptions"
+	"github.com/redhat-developer/odo/pkg/odo/genericclioptions/clientset"
+)
+
+// BindingRecommendedCommandName is the recommended binding sub-command name
+const BindingRecommendedCommandName = "binding"
```

Comment:
It's `RecommendedCommandName` pretty much everywhere in the code. Any reason for naming it differently in this file?

Reply:
Because there's already a constant with that name in the package.